README.md

```diff
@@ -1,2 +1,17 @@
 # HW01
-Edit the readme
+### Andrea Watson
+
+Hi! I'm a rising 5th year PhD candidate in the Committee on Microbiology. Here is my ~~not very professional~~ twitter: **[\@awatsontxt](https://twitter.com/awatsontxt)**
```

Comment:
Thanks for sharing the code for strike through. I believe you are very professional ~~at least in coding~~. ;)